src/main/java/org/example/csv2tex/csv/CsvToSchoolReportDataParser.java

```diff
@@ -106,10 +106,10 @@ private SchoolReportData createBaseData(List<String> rawData) {
         singleStudentData.schoolYear = rawData.get(3);
         singleStudentData.partOfYear = rawData.get(4);
         singleStudentData.birthDay = rawData.get(5);
-        singleStudentData.absenceDaysTotal = rawData.get(6);
-        singleStudentData.absenceDaysUnauthorized = rawData.get(7);
-        singleStudentData.absenceHoursTotal = rawData.get(8);
-        singleStudentData.absenceHoursUnauthorized = rawData.get(9);
+        singleStudentData.absenceDaysTotal = CsvParsingUtil.checkEmptySetDefaultZero(rawData.get(6));
+        singleStudentData.absenceDaysUnauthorized = CsvParsingUtil.checkEmptySetDefaultZero(rawData.get(7));
+        singleStudentData.absenceHoursTotal = CsvParsingUtil.checkEmptySetDefaultZero(rawData.get(8));
+        singleStudentData.absenceHoursUnauthorized = CsvParsingUtil.checkEmptySetDefaultZero(rawData.get(9));
```

Comment:
"sets '0' if it is empty in the input" deserves a test case, imo (or, one of the currently-broken tests would already cover this)

Reply:
It does. I'll take care of it. The tests don't cover it atm.